bh/src/assets/scss/buttons.scss

```diff
@@ -29,6 +29,14 @@
     outline: none;
     box-shadow: .2rem .2rem .5rem rgba(200, 200, 200, .5);
 
+    svg {
```

Comment:
You can try refer `.bh-icon`

Reply:
If i refer .bh-icon it does not have the expected behavior.